include/ninshiki_cpp/node/ninshiki_cpp_node.hpp

```diff
@@ -59,18 +59,18 @@ class NinshikiCppNode
 private:
   using Contours = ninshiki_interfaces::msg::Contours;
   using DetectedObjects = ninshiki_interfaces::msg::DetectedObjects;
-  using Image = shisen_interfaces::msg::Image;
+  using Image = sensor_msgs::msg::Image;
```

Comment:
Please remove the header files related to this change, also the linked libraries.